releasenotes/notes/add-back-noise-simulation-fake-backend-v2-1ec971ac47568e18.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Reverting to the previous behavior. 
+    Add back noisy simulation of Pulse job for FakeBackendV2. 
```

Comment:
This line should be removed now.

Reply:
I think we still need a release note to tell aer >= 0.11.2 (previously, no version restriction for Aer) is now required for users who want noisy simulation with FakeBackendV2.
Regarding title, I would say `Simplify the noise model construction for noisy simulation with FakeBackendV2` or something but you could think of a better one.